nilearn/image/tests/test_image.py

```diff
@@ -271,3 +272,15 @@ def test_swap_img_hemispheres():
         data)
 
 
+def test_concat_imgs():
+    assert_true(concat_imgs is niimg_conversions.concat_niimgs)
+    
+    # simply smoke-test the import from _utils.concat_niimgs
```

Comment:
I guess you don't need this test any more, i.e. all the testing is done in the _utils.niimg_conversions test